src/test/py/peer/peer_test.py

```diff
@@ -113,6 +121,37 @@ async def test_commission_art_piece(self, mock_input):
             await self.deadline_task
             self.assertEqual(self.mock_node.set.call_count, 2)
 
+    def test_add_to_art_collection(self):
+        """
+        Test case for the add_to_art_collection method of the Peer class.
+        This test verifies that the add_to_art_collection method correctly
+        adds an artwork to a collection.
+        """
+        self.peer.add_to_art_collection(self.artwork1, self.collection1)
+        self.assertIn(self.artwork1, self.collection1.get_artworks())
+
+    def test_remove_from_art_collection(self):
+        """
+        Test case for the remove_from_art_collection method of the Peer class.
+        This test verifies that the remove_from_art_collection method correctly
+        """
+        self.collection1.add_to_art_collection(self.artwork1)
+        self.peer.remove_from_art_collection(self.artwork1, self.collection1)
+        self.assertNotIn(self.artwork1, self.collection1.get_artworks())
+
+    def test_swap_art(self):
+        """
+        Test case for the swap_art method of the Peer class.
+        This test verifies that the swap_art method correctly swaps artworks
+        """
+        self.collection1.add_to_art_collection(self.artwork1)
+        self.collection2.add_to_art_collection(self.artwork2)
```

Comment:
Optional/Opportunistic Improvement

I would move the `add_to...` lines to `setUp`, since the other tests rely on it too.